src/test/java/com/wikia/webdriver/testcases/desktop/anonymization/AnonymizationTests.java

```diff
@@ -170,6 +170,12 @@ public void anonymizationMessageWallRemoved() {
 
     Assertion.assertStringContains(anonymizationPage.getAnonConfirmation(), qanon);
 
+    try {
+      Thread.sleep(5000);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this use of "Thread.sleep()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.sjc-dev.k8s.wikia.net/coding_rules#rule_key=squid%3AS2925)

[project-id]: com.wikia.selenium-tests

Reply:
Why sleep instead of waiting for something?

Reply:
Dirty tactical fix. That test already sleeps in other place and needs general refactor.